impl/command_line/options.cxx

```diff
@@ -146,6 +148,8 @@ namespace substrate::commandLine
 
 	[[nodiscard]] std::string_view option_t::metaName() const noexcept
 	{
+		if (_option.valueless_by_exception())
+			return ""sv;
```

Comment:
There's one instance of `""s` in the code in a similar place, I think `string` or `string_view` should be used, unless I'm missing something?

Reply:
This gets to be a string_view because it's non-allocating, etc, etc; the other instance is in a function that returns a std::string because it must allocate on one of the paths, making us have to do so for all